mottak-arkiv-service/tests/routers/dto/test_Arkivuttrekk.py

```diff
@@ -46,8 +47,9 @@ def _arkivuttrekk_dto() -> Arkivuttrekk_dto:
         arkiv_sluttdato=date.fromisoformat("1900-05-12"),
         storrelse=45620,
         avtalenummer="01/12345",
+        depotinstitusjon="Arkivverket",
```

Comment:
så matcher det forventet verdi
```suggestion
        depotinstitusjon="arkivverket",
```